src/ResourceManager/GuestConfiguration/Commands.GuestConfiguration/help/Get-AzureRmVMGuestPolicyReport.md

```diff
@@ -0,0 +1,247 @@
+---
+external help file: Microsoft.Azure.Commands.GuestConfiguration.dll-Help.xml
+Module Name: AzureRM.GuestConfiguration
+online version:
```

Comment:
The ci is failing becasue there is no entry for this field.  You should insert this URI: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/GuestConfiguration/Commands.GuestConfiguration/help/Get-AzureRmVMGuestPolicyReport.md